src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/PoolWriter.java

```diff
@@ -94,6 +95,8 @@
     /** The inner classes to be written, as an ordered set (enclosing first). */
     LinkedHashSet<ClassSymbol> innerClasses = new LinkedHashSet<>();
 
+    HashSet<ClassSymbol> valueClasses = new HashSet<>();
```

Comment:
nitpicking:`Set` instead of `HashSet` for the type definition? (hm, the interface is used below in line 101 but not in the line 96 above)